Fix Typos and Minor Issues in Multiple Files

dsarfed opened the pull request:
This Pull Request addresses various typographical errors and minor issues across multiple files in the project. These corrections enhance code quality, ensure consistency, and prevent potential runtime issues.

#### **What Changed? Why?**
- **`styles.module.css`**
  - Corrected the invalid CSS property `display: hidden;` to `display: none;` in the `.bannerSpacer` class.
  - Removed the typo `dispaly: none;` from the `.bannerSpacer` class.

- **`503.tsx`**
  - Fixed the typo in the error message from `Error has occured` to `Error has occurred`.

- **`BridgeCards.tsx`**
  - Updated the CSS class from `bg-hovergray` to `bg-hover-gray` for consistency and to match naming conventions.

- **`DesktopNav.tsx`**
  - Removed the duplicated `items-center` class from the navigation container to prevent redundant styling.

- **`_document.tsx`**
  - Eliminated duplicate `meta` tags for `twitter:domain` to adhere to HTML standards and avoid potential SEO issues.

cb-heimdall commented:
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>
<!-- CODEOWNERS & Bar Raisers -->
<!-- Power Rangers -->
<!-- Links -->

wbnns commented on apps/base-docs/src/components/Banner/styles.module.css:
```diff
@@ -47,7 +47,7 @@
 
 .bannerSpacer {
-  display: hidden;
+  dispaly: none;
```
Introduces a typo rather than removing

wbnns commented on apps/bridge/pages/_document.tsx:
```diff
@@ -17,7 +17,7 @@ export default function Document() {
         <meta property="og:image" content="https://bridge.base.org/images/base-open-graph.png" />
         <meta name="twitter:card" content="summary_large_image" />
-        <meta property="twitter:domain" content="base.org" />
+        <meta name="twitter:domain" content="base.org" />
```
The correct syntax is property

wbnns reviewed (changes requested):
@dsarfed 

Thanks for this PR! Left a couple of nits in-line.

dsarfed commented:
Hi @wbnns I've addressed the feedback:
- Reverted the `twitter:domain` meta tag to use `property` instead of `name`.
- Corrected the `display` in `styles.module.css`.

wbnns commented:
@dsarfed Could you please rebase and resolve the merge conflict?
